src/transformers/modeling_tf_utils.py

```diff
@@ -2919,21 +2991,43 @@ def from_pretrained(
                     ignore_mismatched_sizes=ignore_mismatched_sizes,
                     tf_to_pt_weight_rename=tf_to_pt_weight_rename,
                 )
+        elif safetensors_from_pt:
+            from .modeling_tf_pytorch_utils import load_sharded_pytorch_safetensors_in_tf2_model
+
+            # Load from sharded PyTorch safetensors checkpoint
```

Comment:
nit - comment isn't really necessary here - the function name is pretty explicit 

```suggestion
```